silk/config.py

```diff
@@ -30,7 +30,8 @@ class SilkyConfig(metaclass=Singleton):
         'SILKY_STORAGE_CLASS': 'silk.storage.ProfilerResultStorage',
         'SILKY_MIDDLEWARE_CLASS': 'silk.middleware.SilkyMiddleware',
         'SILKY_JSON_ENSURE_ASCII': True,
-        'SILKY_ANALYZE_QUERIES': False
+        'SILKY_ANALYZE_QUERIES': False,
+        'SILKY_SENSITIVE_KEYS': {'username', 'api', 'token', 'key', 'secret', 'password', 'signature'}
```

Comment:
this change also need docs for usage

Reply:
thanks, added usage docs under README.md for your review